Gemfile

```diff
@@ -12,8 +12,12 @@ gem 'rails', '~> 7.0.4'
 gem 'sprockets-rails'
 
 # Use sqlite3 as the database for Active Record
-gem 'sqlite3', '~> 1.4'
+# gem 'sqlite3', '~> 1.4'
+gem 'pg'
+<<<<<<< HEAD
 
+=======
+>>>>>>> changes in gemfile, gitignore
```

Comment:
```suggestion

```

This is a leftover from the rebase. Please remove it, otherwise you won't be able to install any gems on this branch :)

Reply:
Check :)